basil/dut.py

```diff
@@ -118,13 +118,13 @@ def catch_exception_on_init(mod):
             except NotImplementedError:
                 pass
 
-        for item in itervalues(self._transfer_layer):
+        for item in self._transfer_layer.values():
```

Comment:
This would degrade Python 2 compatibility, since calls to DUT would be much slower. Big objects would be created on every access to DUT. I would not drop this, just to make it look slightly prettier.